tensorflow/lite/python/convert_test.py

```diff
@@ -68,6 +68,20 @@ def testQuantization(self):
         quantized_input_stats=[(0., 1.)])
     self.assertTrue(tflite_model)
 
+  def testInferenceTypeInvalid(self):
```

Comment:
nit: Please move this test after `testQuantizationInvalid`

Reply:
@gargn thanks alot for spending the time on the review, i have updated as per your comments, kindly approve the same.